src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -312,24 +313,37 @@ def get_job_logs(self, params):
         }
         there are only 100 "log lines" in total.
         """
-        if params["job_id"] == JOB_CREATED:
+        job_id = params["job_id"]
+
+        def log_gen(log_params, total_lines=100):
+            skip = log_params.get("skip_lines", 0)
+            lines = list()
+            if skip < total_lines:
+                for i in range(total_lines - skip):
+                    lines.append(
+                        {"is_error": 0, "line": "This is line {}".format(i + skip)}
+                    )
+            return {"last_line_number": max(total_lines, skip), "lines": lines}
+
+        if job_id == "JOB_LOG_TEST":
+            return log_gen(params, total_lines=100)
```

Comment:
Altered these responses a bit so it's only the `JOB_LOG_TEST` ID that generates 100 lines of log data. Otherwise, completed jobs will return five lines of log data, and any other jobs return the log not found message (to make life easier).